examples/sample-foundry-project/package.json

```
@@ -4,9 +4,9 @@
   "private": true,
   "scripts": {
     "test": "forge test",
-    "build-cannonfile": "npx @usecannon/cli build cannonfile.toml && npx @usecannon/cli build cannonfile.consumer.toml",
-    "run": "npx @usecannon/cli greeter",
-    "run-synthetix": "npx @usecannon/cli synthetix"
+    "build-cannonfile": "node ./node_modules/@usecannon/cli/bin/cannon.js build cannonfile.toml && node ./node_modules/@usecannon/cli/bin/cannon.js build cannonfile.consumer.toml",
+    "run": "node ./node_modules/@usecannon/cli/bin/cannon.js greeter",
+    "run-synthetix": "node ./node_modules/@usecannon/cli/bin/cannon.js synthetix"
```

This change is not necessary, as the `npx` command will prioritise the local `./node_modules` cli over the global one. And you can also just use `cannon`:
```suggestion
    "build-cannonfile": "npx cannon build cannonfile.toml && npx cannon build cannonfile.consumer.toml",
    "run": "npx cannon greeter",
    "run-synthetix": "npx cannon synthetix"
```